diff --git a/node_modules/@react-navigation/elements/lib/module/Header/Header.js b/node_modules/@react-navigation/elements/lib/module/Header/Header.js
index db27170..7cafc2d 100644
--- a/node_modules/@react-navigation/elements/lib/module/Header/Header.js
+++ b/node_modules/@react-navigation/elements/lib/module/Header/Header.js
@@ -244,42 +244,31 @@ export function Header(props) {
     ...props
   }) : customTitle;
   return /*#__PURE__*/_jsxs(Animated.View, {
-    pointerEvents: "box-none",
     style: [{
       height,
       minHeight,
       maxHeight,
       opacity,
       transform
-    }],
+    }, { pointerEvents: 'box-none' }],
     children: [/*#__PURE__*/_jsx(Animated.View, {
-      pointerEvents: "box-none",
-      style: [StyleSheet.absoluteFill, backgroundContainerStyle],
+      style: [StyleSheet.absoluteFill, backgroundContainerStyle, { pointerEvents: 'box-none' }],
       children: headerBackground ? headerBackground({
         style: backgroundStyle
       }) : /*#__PURE__*/_jsx(HeaderBackground, {
-        pointerEvents:
-        // Allow touch through the header when background color is transparent
-        headerTransparent && (backgroundStyle.backgroundColor === 'transparent' || Color(backgroundStyle.backgroundColor).alpha() === 0) ? 'none' : 'auto',
-        style: backgroundStyle
+        style: [backgroundStyle, { pointerEvents: headerTransparent && (backgroundStyle.backgroundColor === 'transparent' || Color(backgroundStyle.backgroundColor).alpha() === 0) ? 'none' : 'auto' }]
       })
     }), /*#__PURE__*/_jsx(View, {
-      pointerEvents: "none",
-      style: {
-        height: headerStatusBarHeight
-      }
+      style: [{ height: headerStatusBarHeight }, { pointerEvents: 'none' }]
     }), /*#__PURE__*/_jsxs(View, {
-      pointerEvents: "box-none",
-      style: [styles.content, Platform.OS === 'ios' && frame.width >= IPAD_MINI_MEDIUM_WIDTH ? styles.large : null],
+      style: [styles.content, Platform.OS === 'ios' && frame.width >= IPAD_MINI_MEDIUM_WIDTH ? styles.large : null, { pointerEvents: 'box-none' }],
       children: [/*#__PURE__*/_jsx(Animated.View, {
-        pointerEvents: "box-none",
         style: [styles.start, !searchBarVisible && headerTitleAlign === 'center' && styles.expand, {
           marginStart: insets.left
-        }, leftContainerStyle],
+        }, leftContainerStyle, { pointerEvents: 'box-none' }],
         children: leftButton
       }), Platform.OS === 'ios' || !searchBarVisible ? /*#__PURE__*/_jsxs(_Fragment, {
         children: [/*#__PURE__*/_jsx(Animated.View, {
-          pointerEvents: "box-none",
           style: [styles.title, {
             // Avoid the title from going offscreen or overlapping buttons
             maxWidth: headerTitleAlign === 'center' ? layout.width - ((leftButton ? headerBackButtonDisplayMode !== 'minimal' ? 80 : 32 : 16) + (rightButton || headerSearchBarOptions ? 16 : 0) + Math.max(insets.left, insets.right)) * 2 : layout.width - ((leftButton ? 52 : 16) + (rightButton || headerSearchBarOptions ? 52 : 16) + insets.left - insets.right)
@@ -287,7 +276,7 @@ export function Header(props) {
             marginStart: 4
           } : {
             marginHorizontal: 16
-          }, titleContainerStyle],
+          }, titleContainerStyle, { pointerEvents: 'box-none' }],
           children: headerTitle({
             children: title,
             allowFontScaling: titleAllowFontScaling,
@@ -296,10 +285,9 @@ export function Header(props) {
             style: titleStyle
           })
         }), /*#__PURE__*/_jsxs(Animated.View, {
-          pointerEvents: "box-none",
           style: [styles.end, styles.expand, {
             marginEnd: insets.right
-          }, rightContainerStyle],
+          }, rightContainerStyle, { pointerEvents: 'box-none' }],
           children: [rightButton, headerSearchBarOptions ? /*#__PURE__*/_jsx(HeaderButton, {
             tintColor: iconTintColor,
             pressColor: headerPressColor,
diff --git a/node_modules/@react-navigation/elements/lib/module/Screen.js b/node_modules/@react-navigation/elements/lib/module/Screen.js
index 8689e62..4544afa 100644
--- a/node_modules/@react-navigation/elements/lib/module/Screen.js
+++ b/node_modules/@react-navigation/elements/lib/module/Screen.js
@@ -47,14 +47,13 @@ export function Screen(props) {
         value: route,
         children: /*#__PURE__*/_jsx(View, {
           ref: headerRef,
-          pointerEvents: "box-none",
           onLayout: e => {
             const {
               height
             } = e.nativeEvent.layout;
             setHeaderHeight(height);
           },
-          style: [styles.header, headerTransparent ? styles.absolute : null],
+          style: [styles.header, headerTransparent ? styles.absolute : null, { pointerEvents: 'box-none' }],
           children: header
         })
       })
