diff --git a/node_modules/@react-navigation/bottom-tabs/lib/module/views/BottomTabBar.js b/node_modules/@react-navigation/bottom-tabs/lib/module/views/BottomTabBar.js
index cb4e310..82ffd39 100644
--- a/node_modules/@react-navigation/bottom-tabs/lib/module/views/BottomTabBar.js
+++ b/node_modules/@react-navigation/bottom-tabs/lib/module/views/BottomTabBar.js
@@ -251,12 +251,10 @@ export function BottomTabBar({
       paddingBottom: tabBarPosition === 'bottom' ? insets.bottom : 0,
       paddingTop: tabBarPosition === 'top' ? insets.top : 0,
       paddingHorizontal: Math.max(insets.left, insets.right)
-    }], tabBarStyle],
-    pointerEvents: isTabBarHidden ? 'none' : 'auto',
+    }], tabBarStyle, { pointerEvents: isTabBarHidden ? 'none' : 'auto' }],
     onLayout: sidebar ? undefined : handleLayout,
     children: [/*#__PURE__*/_jsx(View, {
-      pointerEvents: "none",
-      style: StyleSheet.absoluteFill,
+      style: [StyleSheet.absoluteFill, { pointerEvents: 'none' }],
       children: tabBarBackgroundElement
     }), /*#__PURE__*/_jsx(View, {
       role: "tablist",
